indexing-service/src/test/java/org/apache/druid/indexing/common/actions/SegmentTransactionalInsertActionTest.java

```diff
@@ -92,13 +93,15 @@ private LockResult acquireTimeChunkLock(TaskLockType lockType, Task task, Interv
   }
 
   @Test
-  public void testTransactionalUpdateDataSourceMetadata() throws Exception
+  public void testTransactionalUpdateDataSourceMetadataWithDefaultSupervisorId() throws Exception
```

Comment:
Maybe consider this naming convention for slightly better readability:

```suggestion
  public void test_updateDataSourceMetadata_withDefaultSupervisorId() throws Exception
```